manifests/d/Daytonaio/Daytona/8.84.0/Daytonaio.Daytona.locale.en-US.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Daytonaio.Daytona
+PackageVersion: 8.84.0
+PackageLocale: en-US
+Publisher: Daytonaio
+PublisherUrl: https://github.com/daytonaio/daytona
+PublisherSupportUrl: https://github.com/daytonaio/daytona
+PackageName: Daytona cli

```

Comment:
Package name is Daytona. Daytona is a single binary that has the cli and server in the same binary. This is just called Daytona.